File: rubyStack/RubyFuns/WizardNinjaSamuraiHuman/wizard.rb
require_relative 'human'
class Wizard < Human
   def initialize()
      super(strength, stealth, intelligence, health) #What is inherited
      @intelligence = 25 #Reassigning the inherited value
      @health = 50
   end
   def heal
      @health+=10
   end
   def fireball(obj)
      obj.health=-20
   end

end   
gafton = Wizard.new
# gafton.displayInformation
puts gafton.strength
puts gafton.stealth
puts gafton.intelligence
puts gafton.health

File: C#/OOP/human/wizard.cs
using System;
using human;

namespace human
{
   class Wizard : Human
   {
      public Wizard(string name) : base(name)  
      {
         Name = name;
         health = 50;
         Intelligence = 25;
      }

      public override int Attack(Human target)
      {
         target.health -= (5*this.Intelligence);
         this.health += (5*this.Intelligence);
         Console.WriteLine($"target's health {target.health}; my health {this.health}");
         return target.health;
      }
      public void Heal(Human target)
      {
         target.health += (10*this.Intelligence);
         Console.WriteLine($"new health: {target.health}");
      }
   }
}